src/angle_finder.cpp

```diff
@@ -2,6 +2,7 @@
 #include <prop_follower/PropAngleRange.h>
 #include <cmath>
 #include <ros/console.h>
+#include <string>
```

Comment:
In cord finder a different string library is imported. Can we use the same library in all files?

Reply:
just using std::string now, no need for std_msgs::String currently, removed the include